lib/SIL/Utils/OwnershipUtils.cpp

```diff
@@ -878,6 +845,160 @@ OwnedValueIntroducer swift::getSingleOwnedValueIntroducer(SILValue inputValue) {
   llvm_unreachable("Should never hit this");
 }
 
+// Find all use points of \p guaranteedValue within its borrow scope where
+// \p guaranteedValue is not itself a borrow introducer.
```

Comment:
Rather than use Borrow Introducer, I would prefer if we centralized on BorrowedValue and talk about those introducing the borrow. Consistency I think is going to be important. We should talk over the comments tomorrow.